Mathlib/NumberTheory/NumberField/CanonicalEmbedding/Basic.lean

```diff
@@ -1202,6 +1202,11 @@ abbrev normAtAllPlaces (x : mixedSpace K) : realSpace K :=
 theorem normAtAllPlaces_apply (x : mixedSpace K) (w : InfinitePlace K) :
     normAtAllPlaces x w = normAtPlace w x := rfl
 
+variable (K) in
+theorem continuous_normAtAllPlaces :
```

Comment:
I don't remember if you have a lot of continuity statement to prove, but have you thought about adding various `fun_prop` attribute to simplify the proofs?